lib/apipie/response_description.rb

```diff
@@ -88,11 +88,16 @@ def initialize(method_description, code, options, scope, block, adapter)
 
       @method_description = method_description
 
-      if code.is_a? Symbol
-        @code = Rack::Utils::SYMBOL_TO_STATUS_CODE[code]
-      else
-        @code = code
-      end
+      @code =
+        if code.is_a? Symbol
```

Comment:
I think `if` blocks support `rescue` like ...

```ruby
if cond
  code
rescue => e
  default
else
  default
end
```

Reply:
🤦 - yes, they almost certainly do!  Will tidy up.

Reply:
Actually, no they don't.  You get 
```ruby
rescue => e
  code
else
  other code
end
```

but not:

```ruby
if cond
  code
rescue => e
  other code
else 
  more code
end
```

Reply:
I just thought that's what rubocop was complaining about without looking at the output, sorry for the bad suggestion.